elixir/apps/web/lib/web/session/cookie.ex

```diff
@@ -80,76 +72,4 @@ defmodule Web.Session.Cookie do
   defp encryption_salt do
     Domain.Config.fetch_env!(:web, :cookie_encryption_salt)
   end
-
-  @doc """
-  Mounts the subject for LiveView sockets
-  Reads the per-account cookie from connect_info and authenticates.
-  """
-  def mount_subject(socket, params, _session) do
```

Comment:
This is more for my own edification.  Why was this needed if we already had an `on_mount` live hook setting the subject?

Reply:
It wasn't - I believe this was moved originally from `Web.Auth` where we had live hooks mixed in with all the other functions.